src/lib/viewers/doc/DocLoader.js

```diff
@@ -17,6 +18,12 @@ const VIEWERS = [
         REP: 'pdf',
         EXT: ['gslide', 'gslides', 'odp', 'ppt', 'pptx', 'key']
     },
+    {
+        NAME: 'AutoCAD',
+        CONSTRUCTOR: AutoCADViewer,
+        REP: 'pdf',
+        EXT: ['dwg']
```

Comment:
Just to confirm: Is this the only AutoCAD extension we're supporting?

Reply:
Yeah, there was talk of supporting .dwgzip and .dfx but the latest is that this is the only one